scripts/core/editor3/components/spaces/SpaceComponent.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const SpaceComponent = (props) => {
+    const spanProps = {...{}, ...props};
```

Comment:
I checked babel repl and both `{...{}, ...props}` and `{ ...props}` generate the same output, so it seems you don't need `...{}`